dist/htmlminifier.js

```diff
@@ -1162,6 +1162,14 @@
     return text;
   }
 
+  function uniqueId(value) {
```

Comment:
Hm, I thought it'd be pretty unlikely that random number would collide with user input. But I guess this doesn't hurt and is not in any frequently-executed code.